Assignment1/NetworkManager.swift

```diff
@@ -0,0 +1,38 @@
+//
+//  NetworkManager.swift
+//  Prob
+//
+//  Created by Dar Dar on 04.10.2023.
+//
+
+import Foundation
+
+class NetworkManager {
+    static let shared = NetworkManager()
+    
+    func fetch(completion: @escaping ([User]) -> ()){
```

Comment:
instead of closure type of fetching I recommend to use new feature in ios, async/await for fetching users